lib/v2/icac/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'https://www.icac.org.hk': {
+        _name: '廉政公署',
+        '.': [
+            {
+                title: '新闻公布',
+                docs: 'https://docs.rsshub.app/government.html#xiang-gang-lian-zheng-gong-shu',
+                source: ['/:lang/press/index.html'],
+                target: '/news/:lang',
```

Comment:
```suggestion
                target: '/icac/news/:lang',
```
[script-standard](https://docs.rsshub.app/joinus/script-standard.html) has a typo in radar rules.
[quick-start](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze) has the right format.

Reply:
```javascript
module.exports = {
    'furstar.jp': {
        _name: 'Furstar',
        '.': [
            {
                title: '最新售卖角色列表',
                docs: 'https://docs.rsshub.app/shopping.html#furstar-zui-xin-shou-mai-jiao-se-lie-biao',
                source: ['/:lang', '/'],
                target: '/characters/:lang',
            },
            {
                title: '已经出售的角色列表',
                docs: 'https://docs.rsshub.app/shopping.html#furstar-yi-jing-chu-shou-de-jiao-se-lie-biao',
                source: ['/:lang/archive.php', '/archive.php'],
                target: '/archive/:lang',
            },
            {
                title: '画师列表',
                docs: 'https://docs.rsshub.app/shopping.html#furstar-hua-shi-lie-biao',
                source: ['/'],
                target: '/artists',
            },
        ],
    },
};
```

In the example I noticed that the route root path was not added to target. Should I add it here? Thanks.

Reply:
> ```js
> module.exports = {
>     'furstar.jp': {
>         _name: 'Furstar',
>         '.': [
>             {
>                 title: '最新售卖角色列表',
>                 docs: 'https://docs.rsshub.app/shopping.html#furstar-zui-xin-shou-mai-jiao-se-lie-biao',
>                 source: ['/:lang', '/'],
>                 target: '/characters/:lang',
>             },
>             {
>                 title: '已经出售的角色列表',
>                 docs: 'https://docs.rsshub.app/shopping.html#furstar-yi-jing-chu-shou-de-jiao-se-lie-biao',
>                 source: ['/:lang/archive.php', '/archive.php'],
>                 target: '/archive/:lang',
>             },
>             {
>                 title: '画师列表',
>                 docs: 'https://docs.rsshub.app/shopping.html#furstar-hua-shi-lie-biao',
>                 source: ['/'],
>                 target: '/artists',
>             },
>         ],
>     },
> };
> ```
> 
> In the example I noticed that the `route root path` was not added to `target`. Should I add it here? Thanks.

Oh! I read it backwards, I thought the other one was wrong. Thanks!